packages/beeq/src/components/drawer/_storybook/bq-drawer.stories.tsx

```diff
@@ -68,6 +69,7 @@ const Template = (args: Args) => {
       </div>
       ${!args.noFooter
         ? html`
+            ${args.customFooterDivider ? html`<bq-divider slot="footer-divider" class="mb-m block" />` : nothing}
```

Comment:
Can we use a nicer divider look in the Storybook example? 😄 

```suggestion
            ${args.customFooterDivider ? html`<bq-divider slot="footer-divider" class="mb-m block" stroke-color="stroke--primary" stroke-thickness="1" />` : nothing}
```

![CleanShot 2024-05-17 at 16 17 36@2x](https://github.com/Endava/BEEQ/assets/328492/aeb5dec6-e431-4a6c-924b-644986cc5d82)